src/web/src/app/app.component.html

```diff
@@ -36,26 +36,29 @@ <h1 class="branding__text" *ngIf="!mobileQuery.matches">SvetIT</h1>
     <span class="spacer"></span>
 
     <div class="app-toolbar__lang-select">
-      <button mat-icon-button (click)=select.open()>
+      <mat-select #select class="langSelect" [(ngModel)]="current_lang_" (ngModelChange)="set_language()">
+		<mat-option *ngFor="let lang of languages" [value]="lang">
+		  <span class="{{lang.icon}}"></span> {{ lang.label }}
+		</mat-option>
+	  </mat-select>
+	  <button class="langSelectButton" mat-icon-button (click)=select.open()>
         <mat-icon *ngIf="current_lang_ === undefined">language</mat-icon>
-        <span *ngIf="current_lang_ !== undefined" [class]="current_lang_.icon"></span>
-        <mat-select #select class="langSelect" [(ngModel)]="current_lang_" (ngModelChange)="set_language()">
-          <mat-option *ngFor="let lang of languages" [value]="lang">
-            <span class="{{lang.icon}}"></span> {{ lang.label }}
-          </mat-option>
-        </mat-select>
+        <span *ngIf="current_lang_ !== undefined" [class]="current_lang_.icon"></span>    
       </button>
     </div>
   </ng-container>
   <ng-container *ngIf="authorized">
 	<div class="app-toolbar__user app-toolbar__user-link" (click)="toggleDropDown()">
-      <mat-icon class="app-toolbar__user-icon">person</mat-icon>
+	  <mat-icon *ngIf="activeBadge" class="app-toolbar__user-icon" matBadge="&#8288;" matBadgePosition="after" matBadgeColor="accent" matBadgeSize="small">person</mat-icon>
+	  <mat-icon *ngIf="!activeBadge" class="app-toolbar__user-icon">person</mat-icon>
```

Comment:
Классный компонент ты нашёл. Я и не знал что есть готовый matBadge. Давай тогда под него с бэка вернём вернём кол-во приглашений.
```suggestion
		<mat-icon
			class="app-toolbar__user-icon"
			[matBadgeHidden]="!userNotificationSize"
			[matBadge]="userNotificationSize"
			matBadgePosition="after"
			matBadgeSize="small">person</mat-icon>
```

Reply:
Готово.